src/clr-core/icon/index.ts

```diff
@@ -7,10 +7,10 @@
 import { registerElementSafely } from '@clr/core/common';
 import { CwcIcon } from '@clr/core/icon-shapes';
 
-registerElementSafely('cwc-icon', CwcIcon);
+registerElementSafely('cds-icon', CwcIcon);
```

Comment:
Are we good with the disconnect between `cds-*` and `Cwc*`?

Reply:
Thanks good catch! I'll rename them.